scss/layout/_header.scss

```diff
@@ -0,0 +1,48 @@
+.header {
+    position: relative ;
+    background-color: #F5F3F0;
+    text-align: center;
+    padding: 4em 2em;
+
+    &::after {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        content: "";
+        background: inherit;
+        z-index: -1;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        transform-origin: top right;
+        transform: skewY(-4deg);
+    }
+
+    .profile-pics {
+        width: 160px;
+        clip-path: circle();
+       
+    }
+
+
+
+    p {
```

Comment:
Maybe you could give this paragraph a class, so you style the class? Unless these styles should apply to all paragraphs in the application. If that's the case then this works.

Reply:
No, the style is for that paragraph alone. will give it a class